tests/models.py

```diff
@@ -27,13 +28,20 @@ class Leak(models.Model):
     description = models.TextField()
 
 
+@python_2_unicode_compatible
 class SeaLion(SealableModel):
     height = models.PositiveIntegerField()
     weight = models.PositiveIntegerField()
     location = models.ForeignKey(Location, models.CASCADE, null=True, related_name='visitors')
     previous_locations = models.ManyToManyField(Location, related_name='previous_visitors')
     leak = models.ForeignKey(Leak, models.CASCADE, null=True)
 
+    def __str__(self):
+        return repr(self)
+
+    def __repr__(self):
+        return '<SeaLion %s %s %s>' % (self.id, self.height, self.weight)
```

Comment:
You'll want to wrap that in `str()` because `__repr__` must return `str/bytes` on Python 2 and `str` on Python 3.

Reply:
Forgot to mention that this is required because we have a `__future__.unicode_literals` import aboves which turns this literal into `unicode` on Python 2 and thus require this `str()` wrapping.